ci/run_unittests.sh

```diff
@@ -84,6 +84,7 @@ if [ "x$CVMFS_CACHE_PLUGIN" != "x" ]; then
   CVMFS_CACHE_CONFIG=$(mktemp /tmp/cvmfs-unittests-XXXXX)
   echo "CVMFS_CACHE_PLUGIN_SIZE=1000" > $CVMFS_CACHE_CONFIG
   echo "CVMFS_CACHE_PLUGIN_TEST=yes" >> $CVMFS_CACHE_CONFIG
+  echo "CVMFS_CACHE_PLUGIN_PATH=/tmp/cvmfs_cache_test_dir/" >> $CVMFS_CACHE_CONFIG
```

Comment:
Since the POSIX cache manager has more options than just the base part, I'd suggest to reuse the mechanism from mountpoint.cc to setup the cache manager and quota manager from a set of options represented by an options manager.